package.json

```diff
@@ -135,7 +135,7 @@
     "eslint-plugin-prettier": "^5.0.0",
     "eslint-plugin-react": "7.32.2",
     "eslint-plugin-react-hooks": "4.6.0",
-    "eslint-plugin-simple-import-sort": "^9.0.0",
+    "eslint-plugin-simple-import-sort": "^12.1.0",
```

Comment:
The version after 9.0.0, the rule will sort for both non-types and types alphabetically and not separate sorting them like the version 9.0.0.
Ex:
9.0.0
```
import { 
  abnomal,
  best,
  type Abnormal,
  type Abtest,
  type Best
}
```
And version > 9
```
import {
  abnormal,
  type Abnormal,
  type Abtest,
  best,
  type Best,
```
Personally, I like the first one, but it's just my opinion 😅

Reply:
Hmmm, I also like the first one...